uni/.gitignore

```diff
@@ -128,3 +128,6 @@ app.*.symbols
 
 # Flutter Devtools
 devtools_options.yaml
+
+# Google Service Account Key
+/android/google-service-account-key.json
```

Comment:
[trailing newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) :-P